mobile/pubspec.yaml

```diff
@@ -79,6 +79,16 @@ flutter:
         - asset: fonts/WorkSans.ttf
         - asset: fonts/WorkSans-Italic.ttf
           style: italic
+        # - asset: fonts/WorkSans-Medium.ttf
```

Comment:
?

Reply:
I wanted to add more font weights to the app for a nicer UI, but it will be required more fixed across the app, so I just commented this out.